reactnative/RNPlayground/app/app.tsx

```diff
@@ -48,6 +48,8 @@ const config = {
       path: "",
     },
     Welcome: "welcome",
+    About: "about",
```

Comment:
What is this config dict about?

Reply:
In this `config`, you can define URL schemes to support cross platform deeplinking. 
More info here https://reactnative.dev/docs/linking